src/main/java/sirius/kernel/xml/StructuredOutput.java

```diff
@@ -71,6 +86,18 @@
      */
     StructuredOutput property(@Nonnull String name, @Nullable Object data);
 
+    /**
+     * Convenience method for {@link #property(String, Object)} prepending a namespace.
+     *
+     * @param namespace the namespace
+     * @param name      the name of the property
+     * @param data      the value of the property
+     * @return the output itself for fluent method calls
+     */
+    default StructuredOutput property(@Nonnull String namespace, @Nonnull String name, @Nullable Object data) {
```

Comment:
shoudln't we move this into XMLStructuredOutput as JSON isn't namespace aware..

Reply:
Good point +1